symfony/src/Repository/ArticleRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Article;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * @method Article|null find($id, $lockMode = null, $lockVersion = null)
```

Comment:
This docs are wrong.

Reply:
yes, I was about to ask the same question